app/views/provider_interface/rejections/check.html.erb

```diff
@@ -13,23 +13,53 @@
         <h1 class="govuk-heading-l">Check details and reject application</h1>
       <% end %>
 
-      <%= render RejectionReasons::RejectionReasonsComponent.new(
-        application_choice: @application_choice,
-        reasons: @wizard.to_model,
-        editable: true,
-      ) %>
-
       <% if @interview_cancellation_presenter.render? %>
         <p class="govuk-body"><%= @interview_cancellation_presenter.text %></p>
       <% end %>
 
-      <div class="govuk-warning-text">
-        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-        <strong class="govuk-warning-text__text">
-          <span class="govuk-warning-text__assistive"></span>
-          The candidate will be sent an email to tell them why you rejected their application.
-        </strong>
-      </div>
+        <p class="govuk-body">The candidate will be sent this email:</p>
```

Comment:
Would it make sense to extract this into a separate component, just not the existing `RejectionReasons::RejectionReasonsComponent` one? (There is a fair bit of conditional logic that may be easier to test in a component).

Not for this PR!

Reply:
💯